cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup Canols oil
```

Comment:
Looks like you misspelled "Canola"! Let me know when you've updated the change.

Reply:
Please take another look